requirements.txt

```diff
@@ -1,5 +1,6 @@
 # Everything needed in production
 
+gunicorn==19.9.0
```

Comment:
I think it would be better to move this installation to the Dockerfile instead

Reply:
Sure, I'll fix that